converttodzi.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```

Comment:
Sharp is Node.js's image-processing tool. All it's dependencies are statically-linked, so you won't have to worry bout `brew install imagemagick` or `libdv` etc, it auto-installs and keeps it in node_modules. This was just a temporary setup so we can use this to immediately convert images we want to dzi. Sharp is faster than all image-processing packages out there for node (of the few that exists) and apparently even more performant than graphicsmagick and imagemagick in certain scenarios. that's why I chose it. It's also currently the only package that has all its dependencies statically linked so completely portable.